File: XperienceComunity.TransformableViewsShared/Models/TransformableViewCategoryItem.cs
﻿namespace HBS.Xperience.TransformableViewsShared.Models
{
    public class TransformableViewCategoryItem
    {
        public int TransformableViewCategoryID { get; set; }
        public string TransformableViewCategoryName { get; set; } = string.Empty;
        public int? TransformableViewCategoryOrder { get; set; }
        public int? TransformableViewCategoryParentID { get; set; }
        public int? TransformableViewCategoryRootID { get; set; }
        public string TransformableViewCategoryTitle { get; set; } = string.Empty;
    }
}

File: XperienceComunity.TransformableViewsAdmin/src/Shared/TransformableViewCategoryItem.ts
export interface ITransformableViewCategoryItem {
    transformableViewCategoryTitle: string;
    transformableViewCategoryGuid?: string;
    transformableViewCategoryID?: number;
    transformableViewCategoryLastModified?: Date;
    transformableViewCategoryName?: string;
    transformableViewCategoryParentID?: number | null;
    transformableViewCategoryRootID?: number | null;
    transformableViewCategoryOrder?: number | null;
}
export default interface TransformableViewCategoryItem extends ITransformableViewCategoryItem {
    children?: TransformableViewCategoryItem[]
}